### PR TITLE
added some line in html

### PR DESCRIPTION
**Pull request by Imvishalemp**

### CODE REVIEW

**Review by faisal-eds — APPROVED** [review]

File: ui.apps/src/main/content/jcr_root/apps/empAssessment1/components/pageComponentOne/pageComponentOne.html
Comment (Ganesh-Gavvala) [inline ui.apps/src/main/content/jcr_root/apps/empAssessment1/components/pageComponentOne/pageComponentOne.html:1]: You guys need to use sling model

File: ui.apps/src/main/content/jcr_root/apps/empAssessment1/components/pageComponentOne/pageComponentOne.html [hunk 1]
```diff
@@ -1 +1,2 @@
-<h1>Hello this is from page component one</h1>
\ No newline at end of file
+<h1>Hello this is from page component one</h1>
+<h2>Hi Faisal how are you</h2>
```
Comment (faisal-eds) [inline ui.apps/src/main/content/jcr_root/apps/empAssessment1/components/pageComponentOne/pageComponentOne.html:2]: you need to use proper naming comvention

Comment (Imvishalemp) [inline ui.apps/src/main/content/jcr_root/apps/empAssessment1/components/pageComponentOne/pageComponentOne.html:2]: hyyyyyy

File: ui.apps/src/main/content/jcr_root/apps/empAssessment1/components/pageComponentOne/pageComponentOne.html [hunk 2]
```diff
@@ -1 +1,2 @@
-<h1>Hello this is from page component one</h1>
\ No newline at end of file
+<h1>Hello this is from page component one</h1>
```
Comment (Ganesh-Gavvala) [inline ui.apps/src/main/content/jcr_root/apps/empAssessment1/components/pageComponentOne/pageComponentOne.html:1]: Please let me know if it is finished

**Review by Ganesh-Gavvala — CHANGES_REQUESTED** [review]

**Review by faisal-eds — APPROVED** [review]

**Review by Ganesh-Gavvala — APPROVED** [review]

**Pull request merged**